app/controllers/braintree_webhooks_controller.rb

```diff
@@ -6,7 +6,7 @@ class BraintreeWebhooksController < ApplicationController
   before_filter do
     unless @current_community.braintree_in_use?
       BTLog.error("Received webhook notification even though '#{@current_community.ident}' does not have Braintree in use")
-      render :nothing => true, :status => 400 and return
+      render :nothing => true, :status => 400
```

Comment:
How is the double render error now handled? As this is in before filter, I'd expect the execution of the controller actions continues after this and double renders are possible. In models this is understandable, but how does this work in ActionControllers?